tests/views_integration/Tribe/Events/Views/V2/Partials/Components/Events_Bar/Views/Views_List/__snapshots__/ItemTest__test_render_with_view_class_name_not_equal_to_current_view__1.php

```diff
@@ -0,0 +1,12 @@
+<?php return '<li class="tribe-events-c-view-selector__list-item tribe-events-c-view-selector__list-item--month">
+	<a
+		href="http://test.tri.be/events/month/"
+		class="tribe-events-c-view-selector__list-item-link"
+		data-js="tribe-events-view-link"
+	>
+		<span class="tribe-events-c-view-selector__list-item-icon tribe-common-svgicon tribe-common-svgicon--month"></span>
```

Comment:
Here we're asserting that the `view_class_name` is what we use, right? The two snapshots are identical and I just want to make sure.

Reply:
One snapshot should have an `active` class (when view class name is equal to current view) and one snapshot should not have an `active` class.